src/app/admin-transaction-history/components/TransactionDetailPopUp.tsx

```
@@ -0,0 +1,343 @@
+                  })}
+                </p>
+                <p className="col-span-2 text-lg font-extrabold underline">ผู้ขาย</p>
+                <p className="font-bold text-slate-500">รหัส : </p>
+                <p className="text-slate-500">{transaction?.seller.id}</p>
+                <p className="font-bold text-slate-500">ชื่อ : </p>
+                <p className="text-slate-500">{transaction?.seller?.firstName + " " + transaction?.seller?.lastName}</p>
+                <p className="font-bold text-slate-500">อีเมล : </p>
+                <p className="text-slate-500">{transaction?.seller?.email}</p>
+                <p className="font-bold text-slate-500">เบอร์โทร : </p>
+                <p className="text-slate-500">{transaction?.seller?.phoneNumber}</p>
+                <p className="col-span-2 text-lg font-extrabold underline">ผู้ซื้อ</p>
```

Biome (1.9.4)

[error] This block statement doesn't serve any purpose and can be safely removed.

Standalone block statements without any block-level declarations are redundant in JavaScript and can be removed to simplify the code.
Safe fix: Remove redundant block.


(lint/complexity/noUselessLoneBlockStatements)